canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-$toolbar-icon-label-margin: 8px;
-$toolbar-icon-size: 16px;
-$toolbar-button-height: 40px;
-$toolbar-button-width: 40px;
-$toolbar-button-sm-height: 32px;
-$toolbar-button-sm-width: 32px;
```

Comment:
Double check all the sample toolbars to make sure it still looks the same as production.
<img width="1058" alt="image" src="https://github.com/user-attachments/assets/7d40dd29-0003-4bd8-aaa1-736553f59533">


vs master:
<img width="1048" alt="image" src="https://github.com/user-attachments/assets/f9a22a53-81b4-4a5d-bbc5-955b3027d188">